AstronomyTests/AstronomyTests.swift

```diff
@@ -68,4 +68,32 @@ class AstronomyTests: XCTestCase {
         XCTAssertNil(error)
     }
 
+    func testFetchRover() {
+        
+        let expectation = self.expectation(description: "wait")
+            MarsRoverClient().fetchMarsRover(named: "Curiosity") {rover,error in
+                self.rover = rover
+                self.error = error
+                expectation.fulfill()
+            }
+            wait(for: [expectation], timeout: 10)
+            XCTAssertNil(error)
+            XCTAssertEqual(rover?.name, "Curiosity")
+        }
+    
+    func testFetchImages() {
+        let expectation = self.expectation(description: "wait")
+        
+        testFetchRover()
```

Comment:
Instead of calling another test case, it would be better to set things up from scratch, or prepare a method that returns what you need so it can be used from multiple tests